routers/web/user/home.go

```diff
@@ -618,6 +618,12 @@ func buildIssueOverview(ctx *context.Context, unitType unit.Type) {
 			shownIssues += int(issueCountByRepo[repoID])
 		}
 	}
+	if len(repoIDs) == 1 {
+		repo := showReposMap[repoIDs[0]]
+		if repo != nil {
+			ctx.Data["Repo"] = repo.OwnerName + "/" + repo.Name
```

Comment:
I would suggest to use a longer name instead of `Repo`, to make code can be grep-ed easily, eg: `SingleRepoLink`.

And please use `repo.Link()` instead of appending strings manually:

```
func (repo *Repository) Link() string {
	return setting.AppSubURL + "/" + url.PathEscape(repo.OwnerName) + "/" + url.PathEscape(repo.Name)
}
```